### ISSUE TITLE
Error-first callbacks

### ISSUE THREAD

**Issue by haykam821** [smilefam/SendBird-SDK-JavaScript#93]
Since promises aren't getting added soon (#9 has been closed), it would be helpful to standardize callbacks to have the error first, like [Node.js already does](https://nodejs.org/api/errors.html#errors_error_first_callbacks). This would allow users to easily promisify methods (the only module I could find that allows promisifying error-last callbacks was [`pify`](https://github.com/sindresorhus/pify)), as well as reduce headaches as most developers would expect the error to come first.

Basically, we'd replace this:

```js
channel.sendUserMessage("Hello, world!", (result, error) => {
    // …
});
```

With this:

```js
channel.sendUserMessage("Hello, world!", (error, result) => {
    // …
});
```

**Comment by Alchemist85K** [comment]
@haykam821 
We provide `setErrorFirstCallback()` to change the error to the first parameter.
You can get the error as the first parameter by calling the `setErrorFirstCallback()` with true.
Thank you for your interest in SendBird SDK. :)

**Comment by haykam821** [comment]
@Alchemist85K I haven't had the time to test that, but it seems that would work for my problem. However, [the type definitions](https://github.com/smilefam/SendBird-SDK-JavaScript/blob/v3.0.87/SendBird.d.ts) do not seem to 'document' this functionality.

**Comment by haykam821** [comment]
This issue was fixed in f0769170674a857fd30927f2361ba08fcca23b0e ([added line](https://github.com/smilefam/SendBird-SDK-JavaScript/commit/f0769170674a857fd30927f2361ba08fcca23b0e#diff-7d3e8f141e550e2c2fcf4971790eb988R61))
